[CON-363] feat: Add Timely guide (#111)
amp-labs/docs: src/provider-guides/timely.mdx

---
title: Timely
---

## What's Supported

### Supported Actions

This connector supports:

- [Proxy Actions](/define-integrations/proxy-actions), using the base URL `https://api.timelyapp.com`.

## Before You Get Started

To connect Timely with Ampersand, you will need [Timely Account](https://app.timelyapp.com/join).

Once your account is created, you'll need to create an app in Timely, configure the Ampersand redirect URI within the app, and obtain the following credentials from your app:

- Client ID
- Client Secret

You will then use these credentials to connect your application to Ampersand.

### Create a Timely Account

Here's how you can sign up for a Timely account:

- Go to the [Timely Sign Up page](https://app.timelyapp.com/join).
- Sign up using your preferred method and verify your email.

### Creating a Timely App

Follow the steps below to create a Timely app and add the Ampersand redirect URL.

1. Log in to your [Timely](https://auth.memory.ai/login) account.

2. Click **Devs**.

3. Click the **New Application** button.

> 4. In the **Register New OAuth application** window, enter the **App Name** and a Redirect URL\*\* enter the Ampersand redirect URL: `https://api.withampersand.com/callbacks/v1/oauth`.

Reviewer: ```suggestion
4. In the **Register New OAuth application** window, enter the **App Name**. Enter the Ampersand redirect URL in **Redirect URL**: `https://api.withampersand.com/callbacks/v1/oauth`.
```

5. Click **Submit**.

   ![Timely App Creation](/images/provider-guides/Oauth-timely.gif)

>    You will find the **Client ID** and **Client Secret** in the **Basic Information** section. Note these credentials, as you will need them to connect your app to Ampersand.

Reviewer: ```suggestion
You will find the **Client ID** and **Client Secret** in the **Basic Information** section. Keep these credentials handy to connect your app to Ampersand later.
```
